datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -240,6 +240,11 @@ type Mutation {
     """
     updateDataJob(urn: String!, input: DataJobUpdateInput!): DataJob
 
+    """
+    Update the metadata about a particular DataPlatformInstance
+    """
+    updateDataPlatformInstance(urn: String!, input: DataPlatformInstanceUpdateInput!): DataPlatformInstance
```

Comment:
I would wait to add this until we actually build out the endpoint/resolver for updating a data platform instance. Otherwise someone might see this as an option to query and it would error out

Reply:
I'm ok with that.
The main goal of this PR is to make DataPlatformInstance metadata accessible via GraphQL. Update is not a requirement at this moment.

Addressed in f646ed36d

Reply:
> * The one about using the _mapper helper_ is still pending; I will have a look at some moment during this week.

@sgomezvillamor sounds good! let me know if you have any questions - I'm happy to provide some more examples or collaborate some more on this.

Also the changes to the mapper aren't extremely important, just more code style as we try to move everything to use this mapping helper.